<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>851</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fbfbfb;</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>24</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>119</green>
         <blue>194</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>119</green>
         <blue>194</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>251</red>
         <green>251</green>
         <blue>251</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="HighlightedText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>119</green>
         <blue>194</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #fbfbfb;
border: none;
border-bottom: 1px solid grey;</string>
   </property>
   <widget class="QMenu" name="Registration">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>1. Registration</string>
    </property>
    <addaction name="actionRegistrationCustomer"/>
    <addaction name="actionRegistrationGuarantor"/>
    <addaction name="actionRegistrationLoan"/>
    <addaction name="separator"/>
    <addaction name="actionCalculator"/>
   </widget>
   <widget class="QMenu" name="Search">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>2. Search</string>
    </property>
    <addaction name="actionSearchCustomer"/>
    <addaction name="actionSearchLoan"/>
    <addaction name="separator"/>
    <addaction name="actionSearchGuarantor"/>
    <addaction name="actionSearchCollateral"/>
    <addaction name="actionSearchCounseling"/>
   </widget>
   <widget class="QMenu" name="Repayment">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>3. Repayment</string>
    </property>
    <addaction name="actionRepaymentSingle"/>
    <addaction name="actionRepaymentBatch"/>
   </widget>
   <widget class="QMenu" name="Overdue">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>4. Overdue</string>
    </property>
    <addaction name="actionOverdueRegistration"/>
    <addaction name="actionOverdueManagement"/>
    <addaction name="actionOverdueSearch"/>
   </widget>
   <widget class="QMenu" name="Settings">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>0. Settings</string>
    </property>
    <addaction name="actionSettingsOfficer"/>
    <addaction name="actionSettingsUser"/>
    <addaction name="actionSettingsAssets"/>
   </widget>
   <widget class="QMenu" name="menu5_Report">
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>5. Report</string>
    </property>
    <addaction name="actionPeriodicLoanBalanceReport"/>
   </widget>
   <widget class="QMenu" name="menu6_Personnel">
    <property name="styleSheet">
     <string notr="true">/* QMenu 항목 스타일 */
QMenu::item {
    background-color: #fbfbfb;
    color: black; /* 기본 글자 색은 검정색 */
}

/* QMenu 항목 커서 위 스타일 */
QMenu::item:selected {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}

QMenu::item:hover {
    background-color: #e0e0e0; /* 커서를 올렸을 때 배경을 연한 회색으로 설정 */
    color: blue; /* 커서를 올렸을 때 글자 색을 파란색으로 설정 */
}</string>
    </property>
    <property name="title">
     <string>6. Personnel</string>
    </property>
    <addaction name="actionPersonnelStaffRegister"/>
    <addaction name="actionPersonnelPayroll"/>
    <addaction name="actionPersonnelDownloadReport"/>
   </widget>
   <addaction name="Registration"/>
   <addaction name="Search"/>
   <addaction name="Repayment"/>
   <addaction name="Overdue"/>
   <addaction name="menu5_Report"/>
   <addaction name="menu6_Personnel"/>
   <addaction name="Settings"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Customer</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>:/customer/검색.png</normaloff>:/customer/검색.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/대출 등록.png</normaloff>:/loan/대출 등록.png</iconset>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/대출 검색.png</normaloff>:/loan/대출 검색.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/보증인 검색.png</normaloff>:/loan/보증인 검색.png</iconset>
   </property>
   <property name="text">
    <string>Guarantor Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/담보 검색.png</normaloff>:/loan/담보 검색.png</iconset>
   </property>
   <property name="text">
    <string>Collateral Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/상담 검색.png</normaloff>:/loan/상담 검색.png</iconset>
   </property>
   <property name="text">
    <string>상 담 검 색</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/발송 검색.png</normaloff>:/loan/발송 검색.png</iconset>
   </property>
   <property name="text">
    <string>Counseling Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset>
     <normaloff>:/loan/이자 계산기.png</normaloff>:/loan/이자 계산기.png</iconset>
   </property>
   <property name="text">
    <string>Calculator</string>
   </property>
   <property name="iconText">
    <string>Calculator</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset>
     <normaloff>:/acceptance/수납 일괄등록.png</normaloff>:/acceptance/수납 일괄등록.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset>
     <normaloff>:/acceptance/수납 자료검색.png</normaloff>:/acceptance/수납 자료검색.png</iconset>
   </property>
   <property name="text">
    <string>수 납 자 료 검 색</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>기 간 별 대 출 현 황</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>기 간 별 상 환 예 정 현 황</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>미 수 자 료 현 황</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>연 체 자 료 현 황</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>연 체 자 료 리 스 트</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>대 출 / 수 납 통 계 리 스 트</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>회 사 정 보 관 리</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>영 업 일 관 리
</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>각 종 코 드 관 리</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_30">
   <property name="text">
    <string>사 용 자 관 리</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_32">
   <property name="text">
    <string>자 료 진 단</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_33">
   <property name="text">
    <string>자 료 백 업</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_34">
   <property name="text">
    <string>회 사 정 보 관 리</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_35">
   <property name="text">
    <string>영 업 일 관 리</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_37">
   <property name="text">
    <string>Manage Officer</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_39">
   <property name="text">
    <string>User Management</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_40">
   <property name="text">
    <string>자 료 진 단</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_43">
   <property name="text">
    <string>자 료 백 업</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_45">
   <property name="text">
    <string>작 업 종 료</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionback">
   <property name="text">
    <string>back</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionforward">
   <property name="text">
    <string>forward</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOverdueRegistration">
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOverdueManagement">
   <property name="text">
    <string>Management</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUserManagement">
   <property name="text">
    <string>User Management</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionGuarantor">
   <property name="text">
    <string>Guarantor</string>
   </property>
  </action>
  <action name="actionLoan">
   <property name="text">
    <string>Loan</string>
   </property>
  </action>
  <action name="actionRegistrationCustomer">
   <property name="text">
    <string>Customer</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRegistrationGuarantor">
   <property name="text">
    <string>Guarantor</string>
   </property>
      <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRegistrationLoan">
   <property name="text">
    <string>Loan</string>
   </property>
  </action>
  <action name="actionSearchCustomer">
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actionSearchLoan">
   <property name="text">
    <string>Loan</string>
   </property>
  </action>
  <action name="actionSearchGuarantor">
   <property name="text">
    <string>Guarantor</string>
   </property>
  </action>
  <action name="actionSearchCollateral">
   <property name="text">
    <string>Collateral</string>
   </property>
  </action>
  <action name="actionSearchCounseling">
   <property name="text">
    <string>Counseling</string>
   </property>
  </action>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionRepaymentBatch">
   <property name="text">
    <string>Batch</string>
   </property>
  </action>
  <action name="actionOverdueRegistration_2">
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
  <action name="actionOverdueManagement_2">
   <property name="text">
    <string>Management</string>
   </property>
  </action>
  <action name="actionOverdueSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionSettingsOfficer">
   <property name="text">
    <string>Officer Management</string>
   </property>
  </action>
  <action name="actionSettingsUser">
   <property name="text">
    <string>User Management</string>
   </property>
  </action>
  <action name="actionOverdueRegistration_3">
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
  <action name="actionOverdueManagement_3">
   <property name="text">
    <string>Management</string>
   </property>
  </action>
  <action name="actionCalculator_2">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionRepaymentSingle">
   <property name="text">
    <string>Single</string>
   </property>
  </action>
  <action name="actionSettingsAssets">
   <property name="text">
    <string>Fixed Asset Management</string>
   </property>
  </action>
  <action name="actionPeriodicLoanBalanceReport">
   <property name="text">
    <string>Periodic Loan Balance Report</string>
   </property>
  </action>
  <action name="actionPersonnelStaffRegister">
   <property name="text">
    <string>Staff Register</string>
   </property>
      <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPersonnelPayroll">
   <property name="text">
    <string>Payroll</string>
   </property>
      <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPersonnelDownloadReport">
   <property name="text">
    <string>Download Report</string>
   </property>
      <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
